zerver/views/webhooks/teamcity.py

```diff
@@ -0,0 +1,54 @@
+﻿# Webhooks for external integrations.
+from __future__ import absolute_import
+from zerver.models import get_client
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+import ujson
+
+
+@api_key_only_webhook_view
+@has_request_variables
+def api_teamcity_webhook(request, user_profile, stream=REQ(default='teamcity')):
+
+    payload = ujson.loads(request.body)
+    message = payload['build']
+
+    for property in message['teamcityProperties']:
+        if property['name'] == 'env.BUILD_IS_PERSONAL' and property['value'] == 'true':
+            # This is a personal build, don't announce it.
+            return json_success()
```

Comment:
This behavior should be documented in the integration documentation, otherwise users may try to test the integration with a personal build and think it doesn't work